pkg/server/server.go

```diff
@@ -1220,11 +1220,14 @@ func (s *Server) Expand(ctx context.Context, req *openfgav1.ExpandRequest) (*ope
 	}
 
 	q := commands.NewExpandQuery(s.datastore, commands.WithExpandQueryLogger(s.logger))
-	return q.Execute(ctx, &openfgav1.ExpandRequest{
-		StoreId:              storeID,
-		AuthorizationModelId: typesys.GetAuthorizationModelID(), // the resolved model id
-		TupleKey:             tk,
-		Consistency:          req.GetConsistency(),
+	return q.Execute(
+		typesystem.ContextWithTypesystem(ctx, typesys), 
+		&openfgav1.ExpandRequest{
+			StoreId:              storeID,
+			AuthorizationModelId: typesys.GetAuthorizationModelID(), // the resolved model id
```

Comment:
```suggestion
			AuthorizationModelId: typesys.GetAuthorizationModelID(), // the resolved model id
```

do we need this line?

Reply:
yes, it is not necessary, removed.